App/Containers/ItemDetailsScreen.js

```diff
@@ -79,14 +79,22 @@ class ItemDetailsScreen extends Component {
             </View>
             <Subtitle styleName="md-gutter-top">{item.description}</Subtitle>
             <Title styleName="md-gutter-top">${item.price} / {item.unit}</Title>
-            <QuantitySelector
-        			value={selectedQuantity}
-        			minQuantity={1}
-        			maxQuantity={item.quantity}
-              onChange={this.setQuantity} />
-            <TouchableOpacity style={styles.button} onPress={this.props.addToCart.bind(this, vendor, item, selectedQuantity)}>
-              <Text style={styles.buttonText}>ADD TO CART</Text>
-            </TouchableOpacity>
+            {vendor.accepting_orders &&
+                <QuantitySelector
+            			value={selectedQuantity}
+            			minQuantity={1}
+            			maxQuantity={item.quantity}
+                  onChange={this.setQuantity} />
+            }
+            {vendor.accepting_orders ?
+              <TouchableOpacity style={styles.button} onPress={this.props.addToCart.bind(this, vendor, item, selectedQuantity)}>
+                <Text style={styles.buttonText}>ADD TO CART</Text>
+              </TouchableOpacity>
+            :
+              <View style={{paddingTop: 50, paddingBottom: 50}}>
```

Comment:
Not hugely applicable here, but just FYI we have a `Theme` directory that holds some values that might help with this kind of UI stuff. For isntance `metrics` for stuff like padding 

https://github.com/H-KE/buymeby_new/blob/master/App/Themes/Metrics.js